components/teaching/notes/NoteReader.tsx

```diff
@@ -53,9 +56,10 @@ interface NoteReaderParams {
     verses: VerseType;
     date: string;
     noteId: string;
+    route: RouteProp<MainStackParamList | HomeStackParamList, 'NotesScreen' | 'LiveStreamScreen'>
```

Comment:
Now there's a TS error: `Type '"NotesScreen" | "LiveStreamScreen"' does not satisfy the constraint '"LiveStreamScreen"'.`

I've had a closer look at your code now, so here's my suggestion: just pass a boolean prop to the `<NoteReader/>` component. This will fix the TS error and simply your code a bit.

Reply:
Thanks again Jon. 
Now passing a boolean prop to `<NoteReader/>` from `<LiveStreamScreen/>`

Reply:
Your welcome Igor. Thanks for making the edits so quickly. Great work 👌